tests/helpers/test_docs.py

```diff
@@ -42,7 +42,7 @@ def test_docs_get_func_description() -> None:
     func = IISLogsPlugin.logs
     func_desc = docs.get_func_description(func, with_docstrings=False)
 
-    assert func_desc == "iis.logs - Return contents of IIS (v7 and above) log files. (output: records)"
+    assert func_desc == "(webserver.)iis.logs - Return contents of IIS (v7 and above) log files. (output: records)"
```

Comment:
I still don't understand what this `(webserver.)` part is and I don't see it documented anywhere? What is it and why is it necessary? It's super confusing because I have no idea how to invoke `target-query -f <what goes here?>` now. Do I do `target-query -f '(webserver.)iis.logs'`?

Reply:
#758 originates from this: https://github.com/fox-it/dissect.target/pull/711#issuecomment-2328561977

It's specifically for `__namespace__ = "foo.bar"` cases, the `NamespacePlugin` does not play a role in it nor does subclassing. I thought that was clear with https://github.com/fox-it/dissect.target/pull/1175#discussion_r2152602390

The only purpose of `NamespacePlugin` is grouping of "same type" plugins. It plays no role in the "plugin/namespace tree". There's no need for the `webserver.iis` syntax and it only complicates the plugin calling conventions with no real benefit.